Deployment/complete-solution.json

```diff
@@ -206,7 +206,21 @@
             "TwitterConsumerKey": "[parameters('TwitterConsumerKey')]",
             "TwitterConsumerSecret": "[parameters('TwitterConsumerSecret')]",
             "TwitterOAuthToken": "[parameters('TwitterOAuthToken')]",
-            "TwitterOAuthTokenSecret": "[parameters('TwitterOAuthTokenSecret')]"
+            "TwitterOAuthTokenSecret": "[parameters('TwitterOAuthTokenSecret')]",
+            "APPINSIGHTS_PORTALINFO": "ASP.NETCORE"
+          }
+        },
+        {
+          "name": "appsettings",
+          "type": "config",
+          "apiVersion": "2015-08-01",
+          "dependsOn": [
+            "[resourceId('Microsoft.Web/Sites/', parameters('webSiteName'), 'production')]"
```

Comment:
Are you sure these only get set in the Production slot?
I thought the notation was a bit different for slot settings, but don't have any examples available at this time.

Reply:
That is why I wanted you to review :P

For some weird reason App Insights will only work if it is a slot setting (ask me how I know). I Googled a bit and found this, for instance here: https://stackoverflow.com/questions/40332945/deployment-slot-specific-appsettin-in-arm-template

What do you think?